src/CMakeLists.txt

```diff
@@ -182,7 +183,7 @@ target_include_directories(iidm
         $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/include>
         $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
 )
-target_link_libraries(iidm PUBLIC LibXml2::LibXml2)
+target_link_libraries(iidm PUBLIC LibXml2::LibXml2 Boost::program_options Boost::filesystem Boost::system ${CMAKE_DL_LIBS})
```

Comment:
Boost DLL is a header only library? Why do you need program_options there?
If you do not expose this dependency in the interface, these dependencies should be declared as PRIVATE.

Reply:
> Boost DLL is a header only library?

Yes it is

> Why do you need program_options there?

My bad! Fixed